src/js/views/check-ins/check-in/check-in_views.js

```diff
@@ -12,7 +12,7 @@ const ContextTrailView = View.extend({
   <a class="js-back check-in__context-link">
     {{fas "chevron-left"}}{{ @intl.checkIns.checkInViews.contextTrailView.backBtn }}
   </a>
-  {{fas "chevron-right"}}{{ patient.first_name }} {{ patient.last_name }} - {{formatHTMLMessage (intlGet "checkIns.checkInViews.contextTrailView.checkInCompleted") date=(formatMoment finished_at "LONG" inputFormat="YYYY-MM-DD") }}
+  {{fas "chevron-right"}}{{ patient.first_name }} {{ patient.last_name }} - {{formatHTMLMessage (intlGet "checkIns.checkInViews.contextTrailView.checkInCompleted") date=(formatMoment finishedTs "LONG" inputFormat="YYYY-MM-DD") }}
```

Comment:
as mentioned below we'll have to convert this date to utc first.  I'm pretty sure since the string doesn't specify it'll read it as the local time.  The inputFormat is wrong as well, but I don't know that we'll need it at all.  I _think_ what we'll want to maybe do is add a `utc` flag to the hash like `inputFormat` like `utc=true` and then if `formatMoment` modify the line like:
```js
date = hash.utc ? moment.utc(date, hash.inputFormat) : moment(date, hash.inputFormat)
```

Reply:
This doesn't seem to be clicking for me. If we're always converting the date to utc before rendering, do we still need the `utc` flag?

Reply:
this is what would be converting the date string to utc, so this'd be to display an attribute on the model unmodified that's stored in utc but is not designated utc